.circleci/config.yml

```diff
@@ -334,46 +530,73 @@ jobs:
           # if additional dependencies were added to requirmeents.txt and the docker image hasn't been updated
           # we'd have to install it here at runtime -- which will make things slow, so do yourself a favor and
           # rebuild the docker image! (it automatically pulls the latest requirements.txt on build)
-          source ~/env3.8/bin/activate
+          source ~/env3.6/bin/activate
```

Comment:
Are you sure about this change/

Reply:
This file is generated automatically from `config-2_1.yml` by CircleCI CI. The original line in `config-2_1.yml` that generates this is:
```
source ~/env<<parameters.python_version>>/bin/activate
```
That line, which hasn't been modified in `config-2_1.yml`, is expanded 20 times, one per job. I think that the diff makes this to look like that the Python version has been changed in a job, but you can see a few lines above that this line belongs to different jobs that have been displaced (`j8_cqlsh-dtests-py38-no-vnodes` and `repeated_upgrade_dtest `), and the diff is just messing messing with us.

Reply:
Oh ok, thank you!